go.mod

```diff
@@ -58,3 +58,5 @@ require (
 	google.golang.org/protobuf v1.28.0 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 )
+
+replace github.com/aws/aws-sdk-go => /Users/dnrnd/Downloads/aws-sdk-go
```

Comment:
Is this mostly to avoid committing the custom SDK? It fails all the CI for testing tho.

Reply:
yeah i was trying to avoid that - i figure i'll have to make the tests pass when it's merged into mainline with the updated SDK anyways

Reply:
the tests pass locally 😊

Reply:
actually just going to remove the go.mod from the PR, should make some tests run at least. this package won't build and neither will the bin though.